requirements.txt

```diff
@@ -1,6 +1,5 @@
 certifi==2019.3.9
 chardet==3.0.4
 idna==2.8
-python-dotenv==0.10.1
 requests==2.21.0
-urllib3==1.24.1
+urllib3==1.25.1
```

Comment:
Do we explicitly need this dependency? I think it'll get pulled in when `requests` gets installed

Reply:
`requests` actually pulls in `certifi`, `chardet`, `idna` and `urllib3`. I could take all those out and leave only `requests` in there.

Reply:
Yes, let's do that. We should only specify the top-level requirement, then let it determine which packages to install